yell

```diff
@@ -5,10 +5,14 @@ var cheerio = require('cheerio'),
     path = require('path'),
     _ = require('underscore'),
     get = require('get');
```

Comment:
This `;` should be a `,`, otherwise `request` will be leaked globally rather than scoped by `var`